streaming-commons.cabal

```diff
@@ -65,12 +66,16 @@ library
   else
     build-depends:     unix
 
+  if impl(ghc < 7.7.0)
```

Comment:
This shouldn't be based on the GHC version, but rather on the bytestring version.